### DISCUSSION TITLE
No module named 'django'

### DISCUSSION THREAD

**Discussion by thgeese** [netbox-community/netbox#8842] [Q&A]
Hi, when starting my docker-compose I'm getting this Error in the netbox worker container:

Traceback (most recent call last):
  File "/opt/netbox/netbox/manage.py", line 8, in <module>
    from django.core.management import execute_from_command_line
ModuleNotFoundError: No module named 'django'

source Netbox V 2.10.3
target Netbox V 2.10.4 
Netbox-Docker V 0.27

**Comment by candlerb** [comment]
Well, clearly there's something wrong with your docker-compose file or the docker image it refers to, so where did you get them from?  The current version of Netbox is v3.1.9, and Netbox-docker is 1.6.0, so what you have is very old.

Also, what do you mean by "source" and "target"?

**Comment by thgeese** [comment]
Yes know that it is very old. For that I figure out I have to first reach the Version 2.11.0 and then jump to Version 3.xx I also tried to use the actual netbox-docker 1.6.0 with no luck. Seems that I can not use netbox-docker 1.6 to deploy the very old Version 2.10.x 

The error occurs when dumping my Backup into the image. I used this procedure:

- Clean Up the postgres volume

- start only postgres container
- dump the backup into postgres
- start the rest

**Reply by candlerb** [reply]
And the error occurs just when the container starts, or when you go into the container with `docker exec` or similar?

But your existing netbox 2.10.3 is working correctly inside a container?

**Reply by thgeese** [reply]
yes that Version is OK. Reading/Updating/Backup/Restore all is working. Also so no errors inside the GUI. I can also access single Containers through Portainer or CLI.

**Reply by candlerb** [reply]
Then I think your best bet is to go for latest possible v2.11 (2.11.12?) - at worst you could do this standalone rather than in a container. It will apply migrations and then you can export it again.

Then you can create v3.1.9 in a container, import the database from 2.11.12, and then when you start netbox it should apply the next set of migrations.

**Reply by thgeese** [reply]
sounds like a solution :-) I will give it a try

**Comment by thgeese** [comment]
Got it working with netbox-container 1.2.0 and netbox version 2.11.12. Thx to ryanmerolle for the help on that.
